src/app/App.js

```diff
@@ -12,6 +12,25 @@ export const App = ({options}) => {
    document.querySelector('#helpOption').addEventListener('click', () => {
       showAPopUpScreen(document.querySelector('#helpScreen'))
    });
-}
 
+   // will be replaced with impoert from service/modes
+   const mode = {  
+      name: 'WHO_IS_THAT_POKEMON',
+      title: "Who's that pokemon?",
+      questionType: "image",
+      answerType: "text",
+      answersNumber: 6,
+  }
+  const mode2 = {
+   name: 'WHAT_DOES_THIS_POKEMON_LOOK_LIKE',
+   title: "What does this pokemon look like?",
+   questionType: "text",
+   answerType: "image",
+   answersNumber: 4
+}
+  document.querySelector("#startGameButton").addEventListener("click", function() {
+     renderQuizPage(mode2);
+   });
```

Comment:
Can be changed to arrow function (arrow functions are cool)
```suggestion
  document.querySelector("#startGameButton").addEventListener("click", () => renderQuizPage(mode2));
```